README.md

```diff
@@ -60,7 +60,76 @@ When creating a new entity definition you may use the following files as a guide
 
 For more concrete examples, you can take a look at the files located on the [definitions](./definitions) folder. 
 
-#### Conditions
+#### Synthesis
+
+The definition file may contain a synthesis section which is optional *only* if you can ensure that the telemetry is being stamped with the entity GUID and tags based on rules defined internally at NewRelic.
+
+If the telemetry you want to synthesize entities from always has the same attributes you can define a synthesis section with the following format:
+
+<details>
+  <summary>Example of `synthesis` section with simple format</summary>
+  
+  ```
+  synthesis:
+    # [mandatory] The name of a telemetry attribute that will be used as the name of the entity (i.e. k8s.cluster.name).
+    name: attributeNameA
+    # [mandatory] The name of a telemetry attribute that will be used as the id of the entity, so it needs to be unique. It can be the same field used for the name or not.
+    identifier: attributeNameA
+    # [mandatory] Condition that must be met for this entity to be synthesized.
+    conditions:
+      # The attribute’s value must match the provided value
+      - attribute: attributeName
+        value: value
+    # Set to true if the identifier is expected to be longer than our maximum allowed characters, defaults to false
```

Comment:
Reverse sentence? "When the value of the identifier attribute may be longer than X chars, you should set this to true. "